client/Combatant/StaticCombatantViewModel.ts

```diff
@@ -17,6 +18,7 @@ module ImprovedInitiative {
             this.Initiative = combatant.Initiative();
             this.IsPlayerCharacter = combatant.IsPlayerCharacter;
             this.Tags = combatant.Tags();
+            this.AllowSuggestions = Store.Load(Store.User, "PlayerViewAllowPlayerSuggestions");
```

Comment:
The array for `StaticCombatantViewModel` is initialized in the main tracker view (right [here](https://github.com/dlom/improved-initiative/blob/c544a814dcb6391635d17afcf11ace27ed698a87/client/Encounter/Encounter.ts#L277) in Encounter.ts), so the Store loads settings from the tracker's settings.  By the time they're sent to the player view, they're already immutable in the payload.  I copied the pattern from you just a [few lines below](https://github.com/dlom/improved-initiative/blob/c544a814dcb6391635d17afcf11ace27ed698a87/client/Combatant/StaticCombatantViewModel.ts#L25).  I'll double check if I load the player view on my other machine, but it should work as-is.